linchpin/cli/__init__.py

```diff
@@ -119,6 +119,54 @@ def lp_init(self, providers=['libvirt']):
             self.ctx.log_state('Error: {0}'.format(e))
             sys.exit(1)
 
+    def _write_to_inventory(self, tx_id=None, inv_path=None, inv_format="cfg"):
+        latest_run_data = self._get_run_data_by_txid()
+        if tx_id:
+            latest_run_data = self._get_run_data_by_txid(tx_id)
+        all_inventories = []
+        try:
+            for t_id in latest_run_data:
+                targets = latest_run_data[t_id]["targets"][0]
+                # if there are multiple targets mentioned in pinfile
+                # the multiple inventory files are being generated
+                inv_file_count = 0 if len(targets) > 1 else False
+                for name in targets:
+                    try:
+                        lt_data = targets[name]["inputs"]["layout_data"]
+                        layout = lt_data["inventory_layout"]
+                        i_path = targets[name]["outputs"]["inventory_path"][0]
+                        if inv_path and inv_file_count is not False:
+                            i_path = inv_path + str(inv_file_count)
+                        # t_o -> topology_outputs
+                        t_o = targets[name]["outputs"]["resources"]
```

Comment:
You specify resources here, but other places, topology_outputs. I think there may be a few places that consolidation is necessary still. In my mind, resources > topology_outputs because backward compatibility and length.